src/Plugin/Field/FieldFormatter/StrawberryMediaFormatter.php

```diff
@@ -279,9 +279,11 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
                 $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['dr:uuid']  = $file->uuid();
                 // Used to keep annotations around during edit.
                 // Note: Since View modes are cached, if no change to the NODE this will be served from a cache! mmm.
-                $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['keystoreid'] = WebAnnotationController::getTempStoreKeyName($fieldname,$delta, $nodeuuid);
-                $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['webannotations'] = (boolean)$webannotations;
-                $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['webannotations_tool'] = $webannotations_tool ? $webannotations_tool : 'rect';
+                if ($this->currentUser->hasPermission('view strawberryfield webannotation')) {
```

Comment:
I will discard this because ISSUE-102 has a better option for this.

Reply:
ok, ok for me, I think some spurious pr remain into my RC1